TotalCrossVM/android/app/build.gradle

```diff
@@ -46,6 +45,10 @@ android {
             }
         }
     }
+
+    packagingOptions {
+        pickFirst "**/libskia.so"
```

Comment:
has this been tested with both armv7 and arm64? can you confirm it really picks the correct version?

Reply:
Was tested only in arm64. On the solution: it works but can be modified with further tests in the future. The best solution suggested in the post that I linked at the commit does not work in TotalCross (tested).